src/reducers/reducerFunctions/sharedReducer.js

```diff
@@ -0,0 +1,21 @@
+import {
+  RESET_LOADING,
+  SET_ERROR,
+  SET_LOADING,
+  SET_SUCCESS,
+} from "reducers/constants";
+
+export const sharedReducer = (state, { type, payload }) => {
+  switch (type) {
+    case SET_LOADING:
```

Comment:
when the function is called you can set error to null to clear out the previous error states